v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -289,6 +293,8 @@ private static DataSourceConfiguration getDataSourceConfiguration(JdbcIOWrapperC
     if (!config.dbAuth().getPassword().get().isBlank()) {
       dataSourceConfig = dataSourceConfig.withPassword(config.dbAuth().getPassword().get());
     }
+
+    LOG.info("Final DatasourceConfiguration: " + dataSourceConfig);
```

Comment:
Does this print the DataSourceConfig contents or just the address? SOme log statements i added in the past just print the memory address. 
Even if it prints the address we shouldnt fix it now ig due to timelines. We should however track it.

Reply:
avoid using + in logs.
use the logging structure of coding the string with {} and passing the parameter.

Reply:
This does print the config correctly and not just the address (have verified it in a real migration).
Fixed the `+` to use `{}` in current push.